system_files/shared/usr/share/ublue-os/bluefin-cli/bling.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 if [ "$(basename "$SHELL")" = "bash" ]; then
     #shellcheck disable=SC1091
-    . /usr/share/bash-prexec
+	[ -f "${HOMEBREW_PREFIX}"/etc/profile.d/bash-preexec.sh ] && . "${HOMEBREW_PREFIX}"/etc/profile.d/bash-preexec.sh
```

Comment:
Looking nice! - I'd just add `HOMEBREW_PREFIX=${HOMEBREW_PREFIX:-/home/linuxbrew/.linuxbrew}` as a default for this

Reply:
done!